stack-lts.yaml

```diff
@@ -0,0 +1,17 @@
+resolver: lts-6.13
+
+# NOTE: HIE does not intend to support GHC 7.10.x, the tests fail here.
+packages:
```

Comment:
If we don’t intend to support GHC 7.10, wouldn’t it make more sense to just bump the base lower bound and remove all the CPP to make the exhaustiveness checker? Building with GHC 7.10 while not being able to run using 7.10 doesn’t seem very useful.